bot/exts/utilities/realpython.py

```diff
@@ -32,13 +32,16 @@ def __init__(self, bot: Bot):
 
     @commands.command(aliases=["rp"])
     @commands.cooldown(1, 10, commands.cooldowns.BucketType.user)
-    async def realpython(self, ctx: commands.Context, amount: Optional[int] = 5, *, user_search: str) -> None:
+    async def realpython(self, ctx: commands.Context, amount: Optional[int] = 5, *, user_search: str=None) -> None:
         """
         Send some articles from RealPython that match the search terms.
-
         By default the top 5 matches are sent, this can be overwritten to
         a number between 1 and 5 by specifying an amount before the search query.
         """
+        if (user_search is None):
+            await ctx.send(HOME_URL)
+            return
+    
```

Comment:
I think the linter should complaining about having a whitespace in a blank line.

Reply:
Where is the blank space here?

Reply:
![image](https://user-images.githubusercontent.com/70831061/155468614-b6758203-4c6e-4fc4-9ce1-d0c98dc8fa87.png)

Reply:
You are highlighting a newline character there. If it was a space you should've been able to drag from the right to the left (and not up/down). There's a quite tricky difference.